tests/decorators/test_branch_external_python.py

```diff
@@ -22,18 +22,20 @@
 import pytest
 
 from airflow.decorators import task
-from airflow.utils.state import State
+from airflow.exceptions import DownstreamTasksSkipped
 
 pytestmark = pytest.mark.db_test
 
 
-class Test_BranchExternalPythonDecoratedOperator:
+class TestBranchExternalPythonDecoratedOperator:
     # when run in "Parallel" test run environment, sometimes this test runs for a long time
     # because creating virtualenv and starting new Python interpreter creates a lot of IO/contention
     # possibilities. So we are increasing the timeout for this test to 3x of the default timeout
     @pytest.mark.execution_timeout(180)
-    @pytest.mark.parametrize("branch_task_name", ["task_1", "task_2"])
-    def test_branch_one(self, dag_maker, branch_task_name):
+    @pytest.mark.parametrize(
+        "branch_task_name, skipped_task_name", [("task_1", "task_2"), ("task_2", "task_1")]
```

Comment:
nit (i just prefer the non string from)
```suggestion
        ["branch_task_name", "skipped_task_name"], [("task_1", "task_2"), ("task_2", "task_1")]
```

Reply:
I wasn't even aware that this is possible :D